src/Notifier.js

```diff
@@ -195,6 +196,19 @@ const Notifier = {
         return enabled === 'true';
     },
 
+    setBodyEnabled: function(enable) {
+        if (!global.localStorage) return;
+        global.localStorage.setItem('notifications_body_enabled', enable ? 'true' : 'false');
+    },
+
+    isBodyEnabled: function() {
+        if (!global.localStorage) return true;
+        const enabled = global.localStorage.getItem('notifications_body_enabled');
+        // default to true if the popups are enabled
+        if (enabled === null) return this.isEnabled();
```

Comment:
Why `this.isEnabled()` here but `true` at the top? I'd probably just return the raw setting.